src/sidebarConfig.js

```diff
@@ -103,6 +103,14 @@ const config = {
           label: _('Trial Balance'),
           route: '/report/trial-balance',
         },
+        {
+          label: _('GSTR1'),
+          route: '/report/gstr-1',
+        },
+        {
+          label: _('GSTR2'),
+          route: '/report/gstr-2',
+        },
```

Comment:
These routes should be conditional and should only shop up when country `India` is selected.

Reply:
Good point @ankitsinghaniyaz, I'll do the change.

Reply:
Done 💯